scala-package/init-native/linux-x86_64/pom.xml

```diff
@@ -62,22 +66,24 @@
             <compilerStartOption>-std=c++0x</compilerStartOption>
           </compilerStartOptions>
           <compilerEndOptions>
-            <compilerEndOption>-I${project.basedir}/../../../include</compilerEndOption>
-            <compilerEndOption>${all_includes}</compilerEndOption>
-            <compilerEndOption>${cflags}</compilerEndOption>
+            <compilerEndOption>-I${MXNET_DIR}/include</compilerEndOption>
+            <compilerEndOption>-I${MXNET_DIR}/3rdparty/dmlc-core/include</compilerEndOption>
+            <compilerEndOption>-I${MXNET_DIR}/3rdparty/mshadow</compilerEndOption>
+            <compilerEndOption>-I${MXNET_DIR}/3rdparty/dlpack/include</compilerEndOption>
+            <compilerEndOption>-I${MXNET_DIR}/3rdparty/tvm/nnvm/include</compilerEndOption>
+            <compilerEndOption>-DMSHADOW_USE_MKL=0 -DMSHADOW_USE_CUDA=0</compilerEndOption>
```

Comment:
Why we make them =0, if we build with GPU what will happened?

Reply:
Added a README.md file to describe compiler and linker flags.